app/controllers/mentor/knowledges_controller.rb

```diff
@@ -0,0 +1,48 @@
+class Mentor::KnowledgesController < Mentor::BaseController
+  skip_before_action :authenticate_mentor
+
+  def index
+    @knowledges = Knowledge.order('created_at desc')
+  end
+
+  def show
+    @knowledge = Knowledge.find(params[:id])
+  end
+
+  def create
+    @knowledge = Knowledge.new
+    if params[:commit] == 'ajax'
+      render text: ajax_action
+    else
+      Knowledge.create(knowledge_params)
+    end
+  end
+
+  def new
+    @knowledge = Knowledge.new
+  end
+
+  def edit
+    @knowledge = Knowledge.find(params[:id])
+  end
+
+  def update
+    @knowledge = Knowledge.find(params[:id])
+    @knowledge.updata(knowledge_params)
+  end
+
+  def destroy; end
+
+  private
```

Comment:
__[RuboCop]__

Layout/EmptyLinesAroundAccessModifier: Keep a blank line before and after `private`.


<!-- Comment by [SideCI](https://sideci.com) -->